### PR TITLE
Eric (Yincheng) Gao HW1

### PR DESCRIPTION
**Pull request by ericgao29**
The README.md file has been updated to include a brief biography of myself. Besides, the README.html file (produced by the README.md) is also available and provides a more organized look of the assignment.

P.S. My reflections on the whole process are included in the README.md file.

### CODE REVIEW

**Comment by ericgao29** [comment]
P.S. My reflections on the whole process are included in the README.md file.

File: README.md [hunk 1]
```diff
@@ -1 +1,20 @@
-# hw01
\ No newline at end of file
+# MACS 20500 HW1
```
Comment (chloezhao123) [inline README.md:1]: It's nice to see that you documented your homework clearly with which class it is for! Nice habit!

File: README.md [hunk 2]
```diff
@@ -1 +1,20 @@
...
+
+## Name
+*Eric (Yincheng) Gao*
+
+## Background
```
Comment (chloezhao123) [inline README.md:6]: Your list is very clear. Its indents help organize the logic of the information that you provide!

File: README.md [hunk 3]
```diff
@@ -1 +1,20 @@
...
+  * **Major**: Economics
+  * **Minor(s)**: Statistics, Data Science
+
+## Picture of myself
+![](Picture.jpg){width=150px}
+
+## Github account: ericgao29
+[My Github website](https://github.com/ericgao29)
+
+## Reflections on the process
```
Comment (chloezhao123) [inline README.md:18]: Your reflection is very thorough. It clearly states what kinds of problems you have encountered and solved in this assignment as well as the help you needed to complete it.

File: README.md [hunk 4]
```diff
@@ -1 +1,20 @@
...
+* Problems solved: The picture I wanted to insert here was too large, but I didn't know how to set size for local images at the beginning. The tutorial that helped me solve the problem is the answer of *InspectorSands* on [this stackoverflow website](https://stackoverflow.com/questions/15625990/how-to-set-size-for-local-image-using-knitr-for-markdown).
+* General impression: Markdown is a powerful tool to format data science work. The [guide](https://guides.github.com/features/mastering-markdown/) provided by Github and the cheatsheets handed to us in the first class are both wonderful resources that provide brief introduction to Markdown.
```
Comment (chloezhao123) [inline README.md:34]: In general, I think your assignment is well organized. The theme of it is very coherent. After reading your markdown file, I know a little bit more about you. You clearly demonstrated that you have mastered how to use some basics in R Markdown such as headers, inserting images and URL, italics etc. In general, this is a very nice first assignment.

**Review by chloezhao123 — COMMENTED** [review]
Evaluation
----------------------------------------------------
| Topic                                     | Excellent | Satisfactory | Needs Work |
|----------------------------|-----------|--------------|------------|
| **Coding style**                   |      +     |              |            |
| **Coding strategy**             |      +     |              |            |
| **Presentation**                   |      +     |              |            |
| **Achievement, creativity** |     +      |              |            |
| **Ease of access**               |     +      |              |            |

Remarks:

Some praise:
I think your first homework assignment is great. It is very clean and pretty. It conveys what you want the reader yo know about yourself. The format is very neat with the list indicating your majors and interests. The picture and URL are also inserted correctly. 
Something I learned is that I didn't think of this assignment as a way to introduce myself. It would be a perfect fit for the requirements such as inserting pictures, using list and etc.

**Comment by zxgneil** [comment]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |      x     |              |            |
| **Coding strategy**         |     x      |              |            |
| **Presentation**            |      x     |              |            |
| **Achievement, creativity** |           |      x        |            |
| **Ease of access**          |           |       x       |            |

Remarks:
1. I am glad that you were able to find solutions to problems in encounter with the homework. This ability will come handy for this class! 
1. You have experimented more than 4 ways of using markdown language. Good work! 
1. Really nice way of adjusting picture width! 
1. Your commit history is difficult to access. I understand that you want to put version numbers of your project there, but adding a bit more verbal explanation will help others (and yourself if you go back to the project after a few months) understand your work flow.

**Pull request closed without merging**